api/bin/chainflip-broker-api/src/main.rs

```diff
@@ -54,6 +54,7 @@ pub trait Rpc {
 		destination_asset: RpcAsset,
 		destination_address: String,
 		broker_commission_bps: BasisPoints,
+		boost_fee: Option<BasisPoints>,
```

Comment:
Do we have tests to check that we don't break compatibility between releases @dandanlen? I know some bounder tests use SDK, but I'm not sure if they, for example, use `channel_metadata`, so it would be good to have more explicit/exhaustive tests (ideally using hardcoded commands with curl or something like that).

Reply:
Yeah I was wondering in that case do we want to break compatibility or create a new call, and deprecate the old one

Reply:
I was thinking simply putting boost_fee as the last parameter would preserve comatibility (you can use `null` to indicate that `channel_metadata` should be parsed into `None`).